### PR TITLE
chore: added telemetry for toggle graph view

### PR DESCRIPTION
**Pull request by Harshita-mindfire**
This PR aims to add some telemetry around the toggle of graph view and the default graph theme saved by user

# Dendron Extended PR Checklist

- NOTE: the links don't work. you'll need to go into the wiki and use lookup to find the note until we fix some issues in the markdown export

### Basics

- [x] code should follow [Code Conventions](dev.process.code)
- [x] circular dependency check: make sure your code is not introducing new circular dependencies in plugin-core.  See [Avoiding Circular Dependencies](dev.process.code.best-practices).
- [x] sticking to existing conventions instead of creating new ones
  - eg: [if configuration for utilities are already in one module or package, add future utilities there as well](https://github.com/dendronhq/dendron/pull/1960#discussion_r786228021)

### Extended
- General
  - [x] check whether code be simplified
  - [x] check if similar function already exist in the codebase. if so, can it be re-used?
  - [x] check if this change adversely impact performance
- Operations
  - [x] when shipping this change, will it just work or will it introduce additional operational overhead due to complicated interface or known bugs?
- Architecture
  - [x] check if code is introducing changes on a foundational class or interface. if so, call for design review if needed
    - eg: [making changes to DNode](https://github.com/dendronhq/dendron/pull/2158#pullrequestreview-854689586)

### Basics
TODO: update Airtable
- [ ] if you are adding analytics related changes, make sure the [Telemetry](https://wiki.dendron.so/notes/84df871b-9442-42fd-b4c3-0024e35b5f3c.html) docs are updated

### Extended
- [~] can we track the performance of this change to know if it is _successful_?
  - eg: [see usage for export pod](https://github.com/dendronhq/dendron/pull/2190#pullrequestreview-855715612)

### Basics

- [~] [Write Tests](dev.process.qa.test) 
- [ ] [Confirm existing tests pass](dev.process.qa.test)
- [x] [Confirm manual testing](dev.process.qa.test) 
- [~] Common cases tested
- [~] 1-2 Edge cases tested
- [~] If your tests changes an existing snapshot, snapshots have been [updated](dev.process.qa.test)

### Extended
NA

## Docs
NA

### Extended
NA

## Telemetry
Added a property `defaultTheme` for an existing graph event `GraphThemeChanged`
Added a GraphEvent `GraphViewUsed`.
Added a property `state: string` and `type: string` for event `GraphViewUsed`.

### CODE REVIEW

File: packages/plugin-core/src/components/views/NoteGraphViewFactory.ts [hunk 1]
```diff
@@ -173,6 +173,13 @@ export class NoteGraphPanelFactory {
             });
             break;
           }
+
+          case GraphViewMessageEnum.toggleGraphView: {
+            AnalyticsUtils.track(DENDRON_COMMANDS.SHOW_NOTE_GRAPH.key, {
```
Comment (jonathanyeung) [inline packages/plugin-core/src/components/views/NoteGraphViewFactory.ts:178]: Instead of conflating with the command telemetry, can we use the same 'GraphViewUsed' ("view
 here, not "panel") convention as with the other review?
 
 type: "GraphTypeChanged"
 state: "Local" / "Full"

**Review by jonathanyeung — CHANGES_REQUESTED** [review]

**Review by jonathanyeung — APPROVED** [review]
